DESCRIPTION

```diff
@@ -1,15 +1,15 @@
 Package: dashBio
 Title: Dash components for bioinformatics
-Version: 0.0.11
+Version: 0.1.2
 Authors @R: as.person(c(The Plotly Team <dashbio@plot.ly>))
 Description: Dash components for bioinformatics
 Depends: R (>= 3.0.2), manhattanly
 Imports: dash
-Suggests:
+Suggests: 
```

Comment:
Should this whitespace be here?

Reply:
Can fix that!

Reply:
Fixed in 26f04011389f45d1c94438ff7830fa194454e93c.